From 2fe75771bbe79c440d2164b7cfe40e210accf03d Mon Sep 17 00:00:00 2001
From: Eric Sun <jian.sun@freescale.com>
Date: Mon, 21 Nov 2011 19:09:33 +0800
Subject: [PATCH 0655/2463] ENGR00162709 Add Support for MX6Q Sabre Auto

Code change in none-mach code for support MX6Q Sabre Auto.

Signed-off-by: Eric Sun <jian.sun@freescale.com>
---
 sound/soc/imx/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/imx/Kconfig b/sound/soc/imx/Kconfig
index 9bb8b93d..cc0e2ca 100644
--- a/sound/soc/imx/Kconfig
+++ b/sound/soc/imx/Kconfig
@@ -61,7 +61,7 @@ config SND_SOC_IMX_SGTL5000
 
 config SND_SOC_IMX_CS42888
 	tristate "SoC Audio support for i.MX boards with cs42888"
-	depends on I2C && (MACH_MX6Q_ARM2 || MACH_MX53_ARD)
+	depends on I2C && (MACH_MX6Q_ARM2 || MACH_MX6Q_SABREAUTO || MACH_MX53_ARD)
 	select SND_SOC_CS42888
 	select SND_MXC_SOC_MX2
 	help
-- 
1.7.7.4

